sklearn/pipeline.py

```diff
@@ -50,7 +50,9 @@ class Pipeline(_BaseComposition):
     steps : list
         List of (name, transform) tuples (implementing fit/transform) that are
         chained, in the order in which they are chained, with the last object
-        an estimator.
+        an estimator. Note that uncertainties that are generated
```

Comment:
I think that this either belongs in the class dosctring under Notes, or in the method docstring for predict, below. It seems strange to have it here in steps.